docs/socket-groups.md

```diff
@@ -56,14 +56,19 @@ Every next link in this group gives then another 100% overhead.
 ## 2. Backup
 
 This solution is more complicated and more challenging for the settings,
-and in contradiction to Broadcast group, it costs some penalties.
-
-In this group, only one link out of member links is used for transmission
-in a normal situation. Other links may start being used when there's happening
-an event of "disturbance" on a link, which makes it considered "unstable". This
-term is introduced beside "broken" because SRT normally uses 5 seconds to be
-sure that the link is broken, and this is way too much to be used as a latency
-penalty, if you still want to have a relatively low latency.
+and in contradiction to Broadcast group, it costs some penalties. It has
+also advantages: the overhead for redundancy, which in case of broadcast
+groups is 100% per every next link, in this case it is tried to be kept at
+negligible minimum.
+
+In this group, in a normal situation, only one link out of member links is used
+for transmission, while others are just kept alive (the keepalive message
+is sent over them usually once per 1 second). Other links may start being used
+when there's happening an event of "disturbance" on a link, which makes it
+considered "unstable". This term is introduced beside "broken" because SRT
+normally uses 5 seconds to be sure that the link is broken, and this is way too
+much to be used as a latency penalty, if you still want to have a relatively
+low latency.
```

Comment:
```suggestion
Under normal circumstances, only one of the member links in this group is used
for transmission, while the others are just kept alive (a keepalive message
is sent over these links usually once per 1 second). Other links may start being used
when a "disturbance" event occurs on a link, whereupon the link is
considered to be "unstable". There is a distinction to be made here between "unstable" 
and "broken". SRT normally waits 5 seconds to be sure that a link is broken, which is too
much of a penalty if you want to have a relatively low latency. SRT can react more quickly 
to an "unstable" link condition.
```

Reply:
I kept this, but added some text at the end.